mock_transport/cosmos/create_database_and_collection/1_request.json

```diff
@@ -0,0 +1 @@
+{"uri":"https://mindflavor2.documents.azure.com/dbs/sample/colls","method":"POST","headers":{"authorization":"stripped","x-ms-date":"Mon, 16 Aug 2021 16:34:20 GMT","user-agent":"azsdk-rust-cosmos/0.1.0 (1.56.0-nightly; linux; x86_64)","x-ms-version":"2018-12-31"},"body":[123,34,105,100,34,58,34,112,97,110,122,97,100,111,114,111,34,44,34,112,97,114,116,105,116,105,111,110,75,101,121,34,58,123,34,112,97,116,104,115,34,58,91,34,47,105,100,34,93,44,34,107,105,110,100,34,58,34,72,97,115,104,34,125,125]}
```

Comment:
An upside to using absolute paths over relative ones is that requests can be loaded as-is, without needing to side-load an additional "base URL". This makes it easier to play back, and if the base URL ever needs to be updated it can easily be done by creating a new snapshot or doing a straightforward find-replace.

Both the [VCR Cassette format](https://relishapp.com/vcr/vcr/v/6-0-0/docs/cassettes/cassette-format) and the [HAR format](https://w3c.github.io/web-performance/specs/HAR/Overview.html) use absolute URLs, and I imagine it's for similar reasons.

Reply:
Also I do not think it would be right: the whole URL is the result of the code constructing it so we should **definitely** test it. In the example above, the URL embeds the account (`mindflavor2`), the azure partition (`documents.azure.com`) and the REST object to operate upon (`dbs/sample/colls`). 

Simply specifying `dbs/sample/colls` would not be enough.